app/services/bulk_upload/sales/year2024/row_parser.rb

```diff
@@ -1371,4 +1372,10 @@ def validate_buyer_2_nationality
   def valid_nationality_options
     %w[0] + GlobalConstants::COUNTRIES_ANSWER_OPTIONS.keys # 0 is "Prefers not to say"
   end
+
+  def validate_mortgageused
+    if discounted_ownership? && mortgageused.present? && mortgageused.to_i != 1 && mortgageused.to_i != 2
```

Comment:
I'm reading this as raising an error when:
- The ownership type is discounted 
- Mortgage used is "don't know"
- The year is 24/25

The rules in the ACs suggest we should be raising an error when:
- The ownership type is discounted _or_ shared
- Mortgage used is "don't know"
- _Percentage owned is not 100_
- The year is 24/25

Which suggests to me we also need a validation rule for non-100% shared ownership (similar to that in the 2023 parser).

Have I missed something here?